opencensus/trace/config_integration.py

```diff
@@ -32,17 +25,19 @@ def trace_integrations(integrations, tracer=None):
     integrated = []
 
     for item in integrations:
+        module_name = 'opencensus.ext.{}.trace'.format(item)
+        try:
+            module = importlib.import_module(module_name)
+        except Exception as e:
+            log.warning('Failed to import module: {}'.format(item))
```

Comment:
This might be misleading since the module that failed import was e.g. `opencensus.ext.httplib`, not just `httplib` -- and installing `httplib` wouldn't fix anything.

Why log this twice?

Reply:
Good point!

Regarding logging twice, this was the original behavior which I decided to carry forward. The 1st warning message is OpenCensus level warning (warn the user that certain integration didn't happen), the 2nd message is the actual runtime exception with the call stack (e.g. which is if some underlying package dependency is broken).

Reply:
Fixed the misleading warning message.